src/Types.php

```diff
@@ -221,10 +221,23 @@ public static function isFixedString($str)
         return strpos($str, 'fixedstring(') === 0;
     }
 
+    public static function isSimpleAggregateFunction($str)
+    {
+        return strpos($str, 'simpleaggregatefunction(') === 0;
+    }
+
+    protected function getTypeFromSimpleAggregateFunction(&$type)
+    {
+        if (self::isSimpleAggregateFunction($type)) {
+            $type = trim(strstr($type, ','), ' ,)');
```

Comment:
I'm sorry, I didn't think seriously 
like this `simpleaggregatefunction(max, datetime64(3))`  => `datetime64(3`
It needs to be changed 
```php
$type = substr(trim(strstr($type, ','), ' ,'),0,-1);
```

Reply:
> I'm sorry, I didn't think seriously
> like this `simpleaggregatefunction(max, datetime64(3))` => `datetime64(3`
> It needs to be changed
> 
> ```
> $type = substr(trim(strstr($type, ','), ' ,'),0,-1);
> ```

Maybe `preg_match('/\((.*?)\,\s?(.*)\)/', $type, $matches)` it's not bad idea :)

Reply:
preg_ Match is slow in most cases, so I don't recommend this